ikarus-webapp/src/features/objectives/model/ObjectiveGuard.js

```diff
@@ -18,7 +18,7 @@ ObjectiveGuard = function ObjectiveGuard() {
 
   this._lootTransformPenalty = {
     'guard_objective_reward1': [
-      'IKRS_loot_valuables',
+      'IKRS_loot_valuables'
     ]
```

Comment:
This is a bug fix for guard mission. If a guard kills another guard, that squad will no longer get survive bonuses even if they do survive! However, this here had one too many valuables loot packback, effectively boosting the normal kill bonus.